com.biglybt.BiglyBT.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.biglybt.BiglyBT
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: biglybt-runner
+separate-locales: false
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --persist=.biglybt
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: biglybt
+    buildsystem: simple
+    build-commands:
+      - mv biglybt /app/biglybt
+      - install -dm755 /app/bin
+      - install -Dm644 /app/biglybt/biglybt.desktop /app/share/applications/com.biglybt.BiglyBT.desktop
+      - install -Dm644 /app/biglybt/biglybt.svg /app/share/icons/hicolor/scalable/apps/com.biglybt.BiglyBT.svg
+      - install -Dm644 com.biglybt.BiglyBT.metainfo.xml /app/share/metainfo/com.biglybt.BiglyBT.metainfo.xml
+      - install -m755 biglybt-runner /app/bin/biglybt-runner
+      - desktop-file-edit --set-icon=com.biglybt.BiglyBT /app/share/applications/com.biglybt.BiglyBT.desktop
+      - desktop-file-edit --remove-key=Exec /app/share/applications/com.biglybt.BiglyBT.desktop
+      - desktop-file-edit --set-key=Exec --set-value='biglybt-runner' /app/share/applications/com.biglybt.BiglyBT.desktop
+    build-options:
+      no-debuginfo: true
+    sources:
+      - type: archive
+        url: https://github.com/BiglySoftware/BiglyBT/releases/download/v3.0.0.0/Github_BiglyBT_unix.tar.gz
```

Comment:
Consider adding a stanza for https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

Reply:
@barthalion added it